src/main/java/teammates/common/util/FieldValidator.java

```diff
@@ -444,7 +444,7 @@ public String getInvalidityInfoForCourseName(String courseName) {
     public String getInvalidityInfoForNationality(String nationality) {
         Assumption.assertNotNull("Non-null value expected", nationality);
         if (!NationalityHelper.getNationalities().contains(nationality)) {
-            return String.format(NATIONALITY_ERROR_MESSAGE, nationality);
+            return String.format(NATIONALITY_ERROR_MESSAGE, SanitizationHelper.sanitizeForHtml(nationality));
```

Comment:
If you look at the code for those two methods, the value is sanitized before putting it in the error message.